runtime/doc/vim_diff.txt

```diff
@@ -21,9 +21,9 @@ these differences.
 ==============================================================================
 1. Configuration					    *nvim-configuration*
 
-- Use ".nvimrc" instead of ".vimrc" for storing configuration.
-- Use ".nvim" instead of ".vim" to store configuration files.
-- Use ".nviminfo" instead of ".viminfo" for persistent session information.
+- Use `.nvimrc` instead of `.vimrc` for storing configuration.
+- Use `.nvim` instead of `.vim` to store configuration files.
+- Use `.nviminfo` instead of `.viminfo` for persistent session information.
```

Comment:
> though it's odd it is not mentioned in :help help

Since this PR's scope doesn't seem specific anymore, perhaps you could add this patch to this PR:

```diff
diff --git a/runtime/doc/helphelp.txt b/runtime/doc/helphelp.txt
index ca109cf..5e3db84 100644
--- a/runtime/doc/helphelp.txt
+++ b/runtime/doc/helphelp.txt
@@ -318,11 +318,14 @@ name between two bars (|) eg. |help-writing|.
 When referring to a Vim option in the help file, place the option name between
 two single quotes, eg. 'statusline'
 
+When referring to any other technical term, such as a filename or function
+parameter, surround it in backticks (`), e.g., `~/.path/to/nvimrc`.
+
 
 HIGHLIGHTING
 
 To define a column heading, use a tilde character at the end of the line.
-This will highlight the column heading in a different color.  E.g.
+This will highlight the column heading in a different color.
 
 Column heading~
```